src/components/Header.js

```diff
@@ -154,7 +154,7 @@ const Header = ({ setContext }) => {
       utilityContents={utilityItems}
       navLink={
         <SprkTextInput
-          additionalClasses="docs-header-search sprk-u-Width-100"
+          additionalClasses="docs-header-search"
```

Comment:
The gatsby site would have to be updated to get this styling change. So removing the 100% like this would break it.